app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/SubscriptionStatus.scala

```diff
@@ -17,13 +17,13 @@
 package uk.gov.hmrc.cgtpropertydisposalsfrontend.models
 
 import julienrf.json.derived
-import play.api.libs.json.{Format, JsDefined, JsNull, JsObject, JsResult, JsSuccess, JsUndefined, JsValue, Json, OFormat, OWrites, Reads, Writes}
+import play.api.libs.json.OFormat
 
 sealed trait SubscriptionStatus
 
 object SubscriptionStatus {
 
-  final case object OrganisationUnregisteredTrust extends SubscriptionStatus
+  final case object NonTrustOrganisation extends SubscriptionStatus
 
```

Comment:
I'm getting confused again...is this an organisation which is really a trust but has never enrolled as one with HMRC? That is, a pre-enrollment era trust? If so I think the "OrganisationUnregisteredTrust" is a more reflective name or something like UnenrolledTrust or UnregisteredTrust. NonTrustOrganisation seems to imply that it is an organisation coming in but is not a trust (enrolled or not enrolled).

Reply:
We can't distinguish between legacy trusts and organisations which aren't trusts. The class represents any organisation which doesn't have a trust enrolment - so it could be either one. So perhaps the name before was a bit more accurate